### ISSUE TITLE
Toolbar ellipsis tooltip is confusing

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#4239]
@ElectricFeet in testing had this to say:

> While the tooltip on the block-hover-three-dots says "Open settings menu", clicking on it does not open a settings menu. It gives a menu, one of whose items is "Settings". This is confusing — the tooltip is not descriptive of what it does.

This is true, we should think of something better to say here.

**Comment by amandablum** [comment]
I think "block options" is a better solution here, agreeing with @electricfeet.  This would also create some guidance on separating what is available in this ellipses vs available in block settings.  Block Options implies it is specific to this block, which it is, and that it is different from settings, which it is.

**Comment by danieltj27** [comment]
I agree with @amandablum - `Block Options` seems to make the most sense judging by what that menu actually provides the user.

**Comment by amandablum** [comment]
I think options vs settings could have larger implications in terms of gutenberg lexicon, as I think forward to teaching people the finer points of how to use Gutenberg and that there will be so many different places to affect things. In bringing functionality, we're also complicating. Options implies things to be done to *this block*. Settings implies things done to *this block content*. If we are justifying the two different panels (ellipses and block tab) then there should be a very clear delineation of which does what.   for instance, if we added a "move block" option, that would go under options, not settings.  Showing settings as part of options is a helpful way to lead people to the settings tab and establish the hierarchy of options vs settings. The tooltip can be "change block content settings" which makes the point even clearer.  The only other option that I can see if clearly saying "Block settings" vs "Block Content Settings".

**Comment by karmatosed** [comment]
I think we could just say settings over block. Right now we're debating what the side should say.

**Comment by afercia** [comment]
There are other occurrences of "advanced settings" that should be updated as well.

Specifically, the `aria-label` for the sidebar (2 occurrences) should match the new naming. This is important fo screen reader users so they know the sidebar is named "Block settings" and when using the related button in the menu they understand what it refers to.

Right now, I see the sidebar still identified with the old name:
`<div class="edit-post-sidebar" role="region" aria-label="Editor advanced settings" tabindex="-1">`

Also, I se a few occurrences in `docs/design.md` that should be updated as well. /cc @gziolo 

Re-opening to avoid to open a new issue.

**Comment by abdullah1908** [comment]
Hi @afercia @gziolo ,

Do you think, the title should be changed from **Advanced Settings** to **Block Settings**?
https://github.com/WordPress/gutenberg/blob/master/docs/design.md#advanced-settings

Thanks
